run_brer/run_config.py

```diff
@@ -382,6 +382,7 @@ def __train(self, tpr_file=None, **kwargs):
 
             # noinspection PyUnresolvedReferences
             current_target = context.potentials[i].target
+            current_converged = context.potentials[i].converged
```

Comment:
If you want to log this, that's fine, but
* right now `current_converged` is unused
* if you do want to use it, I suggest taking precaution against `converged` not existing. I.e. `getattr(context.potentials[i], 'converged', some_default_value)`

Reply:
Wait... actually I think I understand what's required for `getattr`: 
```python
current_converged = getattr(context.potentials[i], 'converged', false)
```

Reply:
there is a new thread for this, now...